.github/workflows/regression-tests.yml

```diff
@@ -0,0 +1,120 @@
+name: Regression Tests
+'on':
+  workflow_dispatch:
+    inputs:
+      zkevm_agglayer_commit_id:
+        description: 0xPolygon/agglayer (commit id)
+        required: true
+      zkevm_bridge_service_commit_id:
+        description: 0xPolygonHermez/zkevm-bridge-service (commit id)
+        required: true
+      zkevm_bridge_ui_commit_id:
+        description: 0xPolygonHermez/zkevm-bridge-ui (commit id)
+        required: true
+      zkevm_dac_commit_id:
+        description: 0xPolygon/cdk-data-availability (commit id)
+        required: true
+      zkevm_node_commit_id:
+        description: 0xPolygon/cdk-validium-node (commit id)
+        required: true
+      bake_time:
+        description: bake time (minutes)
+        required: false
+        default: 30
+jobs:
+  deploy_devnet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: '${{ github.event.before }}'
```

Comment:
First time I see this, what is it for? :)

Reply:
artifact, removed !